### DISCUSSION TITLE
WebPushTest.com - Web Push Notifications Demo for iOS Safari

### DISCUSSION THREAD

**Discussion by unamashana** [magicbell/community#132] [Changelog]
Please checkout [WebPushTest.com](https://webpushtest.com) for a demo of web-push notifications for iOS Safari 16.5+. 

Since iOS requires a PWA, there aren't very many demos available yet so we put one together. You can also browse the code if you'd like to try and build this into your app - https://github.com/magicbell-io/webpush-test.

We are working this into the MagicBell inbox and will share a demo soon. Please stay tuned, and in the meantime, send us your feedback or comments. 

Thanks,
Hana

**Comment by moxley01** [comment]
Some relevant links, for extra context:

- [Web-push notifications have arrived on Safari in iOS, but with a catch](https://twitter.com/magicbell_io/status/1666454222034948100)
- [iOS now supports web push notifications, and why you should care](https://www.magicbell.com/blog/ios-now-supports-web-push-notifications-and-why-you-should-care)
